### DISCUSSION TITLE
Add LocationID to visual label editor interface

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#236] [Wish List]
### Symbiota Portal Name

All portals

### Is your feature/tool request related to a problem? Please describe.

_LocationID_ is not currently a field listed in the label editor's visual interface:

![Screen Shot 2022-10-27 at 10 20 14 AM](https://user-images.githubusercontent.com/11181096/198356894-f8f0eba0-cfd2-4629-92d6-ce3b1dbd78b0.png)

**Comment by themerekat** [comment]
When might someone need to print the locationID on a label?

**Comment by ljwalker** [comment]
For collections that store or group specimens by location, this is helpful information to have on the label. It is also just helpful reference information to have readily available when examining physical specimens in a collection.
